### PR TITLE
Token rounding

### PR DESCRIPTION
**Pull request by Kishan-Dhakan**

### CODE REVIEW

File: tests/cli_tests/block_rewards_test.go [hunk 1]
```diff
@@ -168,7 +167,7 @@ func TestBlockRewards(t *testing.T) { // nolint:gocyclo // team preference is to
 		}
 		delta := float64(maxTotalRewardsAndFees - minTotalRewardsAndFees)
 		rewardEarned := endReward - startReward
-		assert.InDeltaf(t, minTotalRewardsAndFees, rewardEarned, delta, "total share difference %d is not within range %d", rewardEarned, delta)
+		require.Equalf(t, minTotalRewardsAndFees, rewardEarned, "total share difference %d is not within range %d", rewardEarned, delta)
```
Comment (sum-it08) [inline tests/cli_tests/block_rewards_test.go:170]: remove delta

Comment (sum-it08) [reply tests/cli_tests/block_rewards_test.go:170]: I guess just require.Equal should be good. its self explanatory.. we can remove message

File: tests/cli_tests/block_rewards_test.go [hunk 2]
```diff
@@ -300,7 +299,7 @@ func TestBlockRewards(t *testing.T) { // nolint:gocyclo // team preference is to
 		}
 		delta := float64(maxTotalRewardsAndFees - minTotalRewardsAndFees)
 		rewardEarned := endReward - startReward
-		assert.InDelta(t, minTotalRewardsAndFees, rewardEarned, delta, "total share difference %d is not within range %d", rewardEarned, delta)
+		require.Equalf(t, minTotalRewardsAndFees, rewardEarned, "total share difference %d is not within range %d", rewardEarned, delta)
```
Comment (sum-it08) [inline tests/cli_tests/block_rewards_test.go:302]: remove delta and having just require.Equal() should be good

File: tests/cli_tests/flaky___flaky_test.go [hunk 3]
```diff
@@ -74,7 +74,7 @@ func Test___FlakyScenariosCommonUserFunctions(t *testing.T) {
 
 		require.Equal(t, allocationID, initialWritePool[0].Id)
 		t.Logf("Write pool Balance after upload expected to be [%v] but was [%v]", 0.5, intToZCN(initialWritePool[0].Balance))
-		require.InEpsilonf(t, 0.5-actualExpectedUploadCostInZCN, intToZCN(initialWritePool[0].Balance), epsilon, "Write pool Balance after upload expected to be [%v] but was [%v]", 0.5, intToZCN(initialWritePool[0].Balance))
+		require.Equal(t, 0.5-actualExpectedUploadCostInZCN, intToZCN(initialWritePool[0].Balance), "Write pool Balance after upload expected to be [%v] but was [%v]", 0.5, intToZCN(initialWritePool[0].Balance))
```
Comment (sum-it08) [inline tests/cli_tests/flaky___flaky_test.go:77]: the equal statement and message are not aligning
in equal you're checking 0.5-actualExpectedUploadCostInZCN and in message its just 0.5

**Review by sum-it08 — CHANGES_REQUESTED** [review]

**Review by stewartie4 — APPROVED** [review]

**Pull request merged**